src/main/scala/org/constellation/domain/redownload/RedownloadService.scala

```diff
@@ -437,10 +460,12 @@ class RedownloadService[F[_]: NonEmptyParallel](
     majorityState: SnapshotsAtHeight,
     redownloadInterval: Int
   ): Boolean =
-    getAlignmentResult(acceptedSnapshots, majorityState, redownloadInterval) match {
-      case AlignedWithMajority => false
-      case _                   => true
-    }
+    if (majorityState.isEmpty) false
```

Comment:
I was thinking about new alignment result like `EmptyMajority`.
We would then have:

```scala
getAlignmentResult(acceptedSnapshots, majorityState, redownloadInterval) match {
        case AlignedWithMajority => false
        case EmptyMajority  => false
        case _  => true
}
```

But maybe explicit check before makes more sense 👍

Reply:
:+1: